src/WpfMath/TexFormulaParser.cs

```diff
@@ -416,15 +448,24 @@ internal static SourceSpan ReadElement(SourceSpan value, ref int position)
                         if (position == value.Length)
                             throw new TexParseException("`left` command should be passed a delimiter");
 
-                        var delimiter = value[position];
-                        ++position;
+                        string delimiter = ReadArgument(value, ref position).ToString().Trim();
+
                         var left = position;
 
                         var internals = ParseUntilDelimiter(value, ref position, formula.TextStyle, environment);
 
-                        var opening = GetDelimiterSymbol(
-                            GetDelimeterMapping(delimiter),
+                        SymbolAtom opening = null;
+                        if (delimiter.Length == 1)
+                        {
+                            opening = GetDelimiterSymbol(
+                            GetDelimeterMapping(delimiter[0]),
                             value.Segment(start, left - start));
+                        }
+                        if (delimiter.Length > 1)
+                        {
+                            opening = GetDelimiterSymbol(
+                            delimiter.Replace('\\', ' ').Trim(), value.Segment(start, left - start));
+                        }
```

Comment:
@ForNeVeR please take a close look at this source fragment. I believe it could be improved somehow (at least with `if/else if`).